src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
for tests to run, all d15 dlls need to be in the same directories. since we have this mixed d14 and d15 references all over the places which comes from p2p references, if d14 ends up the last one that copy over to the binary directories. test will fail since it will looking for d15 dlls.

this was a way to make sure d15 dlls are the last one left in the directory. if you think that won't happen without this, or have a better way to ensure it. go ahead.